opslevel/resource_opslevel_service_relationship.go

```diff
@@ -0,0 +1,238 @@
+package opslevel
+
+import (
+	"context"
+	"fmt"
+	"slices"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/opslevel/opslevel-go/v2024"
+)
+
+var _ resource.ResourceWithConfigure = &ServiceRelationshipResource{}
+
+var _ resource.ResourceWithImportState = &ServiceRelationshipResource{}
+
+func NewServiceRelationshipResource() resource.Resource {
+	return &ServiceRelationshipResource{}
+}
+
+// ServiceRelationshipResource defines the resource implementation.
+type ServiceRelationshipResource struct {
+	CommonResourceClient
+}
+
+// ServiceRelationshipResourceModel describes the Service managed resource.
+type ServiceRelationshipResourceModel struct {
+	Id      types.String `tfsdk:"id"`
+	Service types.String `tfsdk:"service"`
+	System  types.String `tfsdk:"system"`
+}
+
+func NewServiceRelationshipResourceModel(service *opslevel.Service, givenModel ServiceRelationshipResourceModel) ServiceRelationshipResourceModel {
+	return ServiceRelationshipResourceModel{
+		Id:      ComputedStringValue(string(service.Id)),
```

Comment:
I don't see any usages of this field - why is it needed?

Reply:
`Id` is being used during import and testing

Reply:
We don't need `Id` 😁 - just dropped this field ✅